### PR TITLE
Newsletter subscription placeholder

### PR DESCRIPTION
**Pull request by dipti2jcommerce**
### Description (*)
On ipad view newsletter subscription input box not contain complete text (placeholder) #21592

### Fixed Issues (if relevant)
On ipad view newsletter subscription input box not contain complete text (placeholder) #21592

### Manual testing scenarios (*)
2019-03-06_13-25_advanced search

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @dipti2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=21593) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=21593) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: dipti2jcommerce<br/>:white_check_mark: p-bystritsky<br/>:x: Dipti 2Jcommerce<hr/>**Dipti 2Jcommerce** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less [hunk 1]
```diff
@@ -90,3 +90,12 @@
...
 }
+
+//
+//  Tablet
+//  _____________________________________________
+@media only screen and (max-width: @screen__m) {
+  .block.newsletter {
+    width: 42%;
+  }
+}
```
Comment (zanilee) [inline app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less]: Hi, @dipti2jcommerce!

Please add an empty line in the end of the file

Thank you!

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21593.instances.magento-community.engineering).
Admin access: [https://pr-21593.instances.magento-community.engineering/admin](https://pr-21593.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
Hi @dipti2jcommerce. Thank you for you contribution. 
Unfortunately, I'm still able to reproduce the issue

image

Please, take a look once again. Thanks

**Comment by dipti2jcommerce** [comment]
Hi @rogyar ,

I have made modification,please check.

Thank,
Dipti

File: app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less (hunk 1 above)
Comment (dipti2jcommerce) [reply app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less]: Hi @zanilee ,

I added empty line in the end of the file.
please check.

Thank,
Dipti

File: app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less [hunk 2]
```diff
@@ -90,3 +90,19 @@
...
+//
+//  Tablet
+//  _____________________________________________
+@media only screen and (max-width: @screen__m) {
+  .footer .block.newsletter {
+    width: 42%;
+  }
+}
+
+
```
Comment (rogyar) [inline app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less]: Please, remove the extra empty lines. It's enough to have just one empty line at the end. Thank you

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less (hunk 2 above)
Comment (dipti2jcommerce) [reply app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less]: Hi @rogyar ,
I have remove extra line from app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less
Thanks,
Dipti

File: app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less [hunk 3]
```diff
@@ -102,3 +112,9 @@
         }
     }
 }
+
+
```
Comment (rogyar) [inline app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less]: Please, remove the extra lines here as well

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less (hunk 3 above)
Comment (dipti2jcommerce) [reply app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less]: Hi,
I have remove the extra lines from app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less.
Please check.

Thanks,
Dipti

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21593.instances.magento-community.engineering).
Admin access: [https://pr-21593.instances.magento-community.engineering/admin](https://pr-21593.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21593.instances.magento-community.engineering).
Admin access: [https://pr-21593.instances.magento-community.engineering/admin](https://pr-21593.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
On some resolutions a part of the placeholder is still hidden. However, on iPads it looks much better after the current fix

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-4683](https://jira.corp.magento.com/browse/ENGCOM-4683) has been created to process this Pull Request

**Comment by VasylShvorak** [comment]
:heavy_check_mark: QA passed

**Comment by sidolov** [comment]
Hi @dipti2jcommerce  , looks like you made some commits with email different than in your GitHub profile, please, add email from commits to your profile!
Thank you!

**Comment by dipti2jcommerce** [comment]
Hi @sidolov ,

I have updated my email address ,current mail id is dipti@twojay.in.

Thanks,
Dipti

**Comment by sidolov** [comment]
Hi @dipti2jcommerce , pull request still contains commits with email different than in your GitHub account. Please, add email from commits to your account

**Comment by rogyar** [comment]
I'm closing this PR because of inactivity. Please, feel free to reopen it if you decide to proceed to work.

**Pull request closed without merging**